cmd/contrib/main.go

```diff
@@ -118,22 +118,36 @@ func newPlugin() error {
 					return suggestions
 				},
 			},
-			Validate: func(ans any) error {
+			Validate: func(ans interface{}) error {
```

Comment:
Can you elaborate on why you changed from `any` to `interface{}`?

Reply:
I remember[ there was a PR](https://github.com/1Password/shell-plugins/pull/73) to replace `any` with an `interface{}` in all the places in the code.
Looks like there are some compatibility issues with the closed-source codebase.

Reply:
In a [later MR](https://github.com/1Password/shell-plugins/pull/102), we've reverted the revert. `any` is now again what we use.

The reason it was initially reverted is because we still had a runner using a version of Go that does not yet support `any`. Since updating that runner to Go 1.19, we have reverted the revert again and switched back to using `any`.

Reply:
Ok. Then I can use `any` there 😄